config/v3_3_experimental/config.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/coreos/ignition/v2/config/merge"
 	"github.com/coreos/ignition/v2/config/shared/errors"
 	"github.com/coreos/ignition/v2/config/util"
+	"github.com/coreos/ignition/v2/config/v3_2"
```

Comment:
I'd be tempted to import this under a name like `prev`.  That way, when we create a new experimental config, we only have to change the import, and don't need to touch the `ParseCompatibleVersion` code.